pkg/security/api/validation/validation_test.go

```diff
@@ -23,12 +30,13 @@ func validPodSpec() kapi.PodSpec {
 		DNSPolicy:             kapi.DNSClusterFirst,
 		ActiveDeadlineSeconds: &activeDeadlineSeconds,
 		ServiceAccountName:    "acct",
+		SchedulerName:         kapi.DefaultSchedulerName,
 	}
 }
 
 func invalidPodSpec() kapi.PodSpec {
 	return kapi.PodSpec{
-		Containers:    []kapi.Container{{}},
+		Containers:    []kapi.Container{{TerminationMessagePolicy: kapi.TerminationMessageReadFile}},
```

Comment:
We can undo and re-test and re-find the path that was failing. But it was definitely failing validation.